packages/swing-store/src/snapStore.js

```diff
@@ -98,16 +110,14 @@ export function makeSnapStore(
 ) {
   db.exec(`
     CREATE TABLE IF NOT EXISTS snapshots (
+      vatID TEXT,
+      endPos INTEGER,
+      inUse INTEGER,
```

Comment:
I added `uncompressedSize`.  I'm wondering if I should also add `compressedSize` to the table while I'm at it, since it would allow this to be queried without requiring fetching the whole (possibly very large) snapshot blob.

I don't like `used` because it has a past tense meaning ("was used"), whereas what we really want to indicate "is in use right now".